src/common/components/clusterWizard/ClusterWizardStepValidationsAlert.tsx

```diff
@@ -36,22 +38,31 @@ const ClusterWizardStepValidationsAlert = <ClusterWizardStepsType extends string
   wizardStepsValidationsMap,
   children,
 }: ClusterWizardStepValidationsAlertProps<ClusterWizardStepsType>) => {
-  const { failedClusterValidations } = React.useMemo(() => {
+  const { failedClusterValidations, failedHostnameValidations } = React.useMemo(() => {
     const reducedValidationsInfo = getWizardStepClusterValidationsInfo(
       validationsInfo || {},
       currentStepId,
       wizardStepsValidationsMap,
     );
+
     const flattenedValues = lodashValues(reducedValidationsInfo).flat();
+
+    const hostnameValidations = hosts.some((host) => {
+      const validations =
+        typeof host.validationsInfo === 'string'
+          ? stringToJSON<HostValidationsInfo>(host.validationsInfo)
+          : host.validationsInfo;
+
+      return checkHostValidations(validations || {}, ['hostname-valid', 'hostname-unique']);
```

Comment:
`hostnameValidations` is true if at least one host is valid, false if all the hosts are valid.
Shouldn't we be looking at the opposite, to have the message appear even if only 1 host is invalid?

Then it should be:
```suggestion
    const hasFailedHostnameValidations = hosts.some((host) => {
       ...
      return !checkHostValidations(validations || {}, ['hostname-valid', 'hostname-unique']);
    });
```